Taskfile.yml

```diff
@@ -5,7 +5,7 @@ env:
   GORACE: halt_on_error=1,history_size=2
 
 vars:
-  INTEGRATIONTIME: 20m
+  INTEGRATIONTIME: 25m
```

Comment:
Oh, this one is getting bigger and bigger...

Reply:
Resolving that conversation because it is the only thing that stops this PR to be auto-merged and unblock other PRs